pkgs/development/interpreters/klong/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, stdenv
+, fetchurl
+, makeWrapper }:
+
+
+
+stdenv.mkDerivation rec {
+  name = "klong";
+  version = "20221212";
+
+  src = fetchurl {
+    url = "https://t3x.org/klong/${name}${version}.tgz";
+    sha256 = "sha256-2IuD3sanS4Rmq0CnBqWTJWx7SJdz2EMdn+8uKhBAC7c=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    install -m 555 -Dt $out/bin kg kplot
+  '';
+
+  fixupPhase = ''
+    mkdir -p $out/lib
+    cp lib/*.kg $out/lib
+    wrapProgram $out/bin/kg --prefix KLONGPATH $out/lib
+  '';
+
+  meta = with lib; {
```

Comment:
```suggestion
  meta = {
```

Add `lib` below.

Reply:
i'm having trouble with this one, where do add `lib` again?

Reply:
got it, thank you.